packages/bff/src/model/api/apiConverter.ts

```diff
@@ -167,3 +167,19 @@ export function toTenantWithOnlyAttributes(
     attributes: tenant.attributes.map(toTenantAttribute).flat(),
   };
 }
+
+export function toCatalogCreateEServiceSeed(
+  eServiceSeed: bffApi.EServiceSeed
+): catalogApi.EServiceSeed {
+  return {
+    ...eServiceSeed,
+    descriptor: {
+      description: "NONE",
```

Comment:
shouldn't this be an empty string?

Reply:
[a6fbb01](https://github.com/pagopa/interop-be-monorepo/pull/885/commits/a6fbb017afcdbe98e159358fdcfd37aa56df0eba)